database/factories/MessageFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Message;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Support\Str;
+
+class MessageFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Message::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'user_id' => User::all()->random(),
```

Comment:
(suggest)
これだと何かしらのユーザーが存在する前提になりませんか？（MessageFactory を実行する前に必ず User が DB に存在することを MessageFactory の利用者が意識しないといけない）

以下のようにすれば、MessageFactory 実行時に勝手に User も作成されます。
```suggestion
            'user_id' => User::factory(),
```